src/commands/node/chopsticks/start.ts

```diff
@@ -0,0 +1,38 @@
+import { Flags } from "@oclif/core";
+import { execaCommand } from "execa";
+import { SwankyCommand } from "../../../lib/swankyCommand.js";
+import { swankyNodeCheck } from "../../../lib/index.js";
+import { pathExistsSync } from "fs-extra/esm";
+import { ConfigError, FileError } from "../../../lib/errors.js";
+export class StartChopsticks extends SwankyCommand<typeof StartChopsticks> {
+  static description = "Start chopsticks";
+
+  static flags = {
+    "config": Flags.string({
+      description: "Path to the chopsticks config file",
+    })
+  }
+
+  async run(): Promise<void> {
+    const { flags } = await this.parse(StartChopsticks);
+
+    swankyNodeCheck(this.swankyConfig);
+
+    if(!this.swankyConfig.node.chopsticks && !flags.chopsticksConfigPath) {
```

Comment:
`chopsticksConfigPath` is not a valid flag, use `chopsticksConfigPath ` previously created